[ENG-49]

cykonetic opened the pull request:
Adds correct address.
Local file paths/names are configurable, but no method to transfer the shared object or executable.

cykonetic committed:
First commit of Expirian CorrectAddress

cykonetic committed:
Code complete, now for testing

cykonetic committed:
ready for integration testing

cykonetic committed:
Merge branch 'master' into ENG-49

cykonetic committed:
Merge branch 'master' of https://github.com/TheDMSGroup/mautic-enhancer into ENG-49

cykonetic committed:
Merge branch 'ENG-49' of https://github.com/TheDMSGroup/mautic-enhancer into ENG-49

cykonetic committed:
Command tests failing, not sure why

cykonetic committed:
[ENG-49] minor tweak

cykonetic committed:
[ENG-49] Updated composer file and more testing

cykonetic committed:
[ENG-49] Removing testing change to bypass 2.2GB download

cykonetic committed:
[ENG-49] yeah...if i dont .gitignore my changes, this might work

cykonetic committed:
[ENG-49] phpstan still not finding flysystem files

cykonetic committed:
[ENG-49] and the answer is ... travis

cykonetic committed:
[ENG-49] if at first you do not succede..

cykonetic committed:
[ENG-49] my style was rejected :-(

heathdutton committed:
Automatically include plugin dependencies.

heathdutton commented on Command/UpdateCorrectAddressDataCommand.php:
```diff
@@ -0,0 +1,109 @@
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        try {
+            echo 'Starting Expirian data update.'.PHP_EOL;
```
Please use the output interface instead of "echo"

heathdutton replied:
`$output->write('<info>Starting Expirian data update</info>');`

heathdutton committed:
Delete composer.lock

File is not needed as the parent project contains it's own lock file, and so this can be gitignored.

heathdutton committed:
Update .gitignore

heathdutton reviewed (changes requested):
Please update the readme to include instructions for installation and configuration, cron job to add, etc.

heathdutton committed:
Merge branch 'master' into ENG-49

Commit:
[ENG-49] Correct Address setup dodocs

Commit:
[ENG-49] Correct Address setup Docs (changes added)

Commit:
[ENG-49] Replaces echos with $output->writelns

heathdutton committed:
Merge branch 'master' into ENG-49
